api/search.go

```diff
@@ -31,7 +31,7 @@ var (
 	err error
 )
 
-func (api *SearchAPI) getSearch(w http.ResponseWriter, r *http.Request) {
+func (api *SearchAPI) getSearch(w http.ResponseWriter, r *http.Request, hasAuth bool) {
```

Comment:
Im not sure how `hasAuth` is being passed into the getSearch method from api.go? Also whether it is needed?

`hasAuth` exists within `api` interface and can be accessed by calling `api.privateAuth.HasPrivateEndpoints`